### ISSUE TITLE
Phase error

### ISSUE THREAD

**Issue by kc9jud** [e-eight/chime#8]
It appears a phase error has been introduced in 2641065da5c25ee02b09762184af4cd6c29443f7.

`relativecm-gen` input `relcm.in`:
```
1 0 1 1
4 15.0
1.0
mm nlo 2
LENPICchi2bM11C_Nmax04_hw15.0_relcm.dat
```

Before:
```
# RELATIVE-CM LSJT
#   version
#   J0 g0 T0_min T0_max symmetry_phase_mode  [P0=(-)^g0]
#   Nmax
#   T0   Nr' lr' Nc' lc' L' S' J' T' g'   Nr lr Nc lc L S J T g   JT-RME
 1
 1 0 1 1 0
 4
   1      1   1   1   1   0   0   0   0   0        1   1   1   1   0   1   1   1   0      -1.98073763e-01
   1      1   1   1   1   0   0   0   0   0        1   1   3   1   0   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        3   1   1   1   0   1   1   1   0      -4.16307108e-02
   1      1   1   3   1   0   0   0   0   0        1   1   1   1   0   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        1   1   3   1   0   1   1   1   0      -1.98073763e-01
   1      1   1   3   1   0   0   0   0   0        3   1   1   1   0   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   1   1   0   1   1   1   0      -4.16307108e-02
   1      3   1   1   1   0   0   0   0   0        1   1   3   1   0   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        3   1   1   1   0   1   1   1   0      -1.14243860e-01
   1      1   1   1   1   0   0   0   0   0        2   2   2   2   1   0   1   1   0      -0.00000000e+00  <<<
   1      1   1   3   1   0   0   0   0   0        2   2   2   2   1   0   1   1   0      -8.73909393e-02  <<<
   1      3   1   1   1   0   0   0   0   0        2   2   2   2   1   0   1   1   0      -0.00000000e+00  <<<
   1      1   1   1   1   0   0   0   0   0        1   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        1   1   3   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        3   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        1   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        1   1   3   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        3   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   3   1   1   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        3   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        1   1   1   1   2   1   1   1   0      2.07579903e-01
   1      1   1   1   1   0   0   0   0   0        1   1   3   1   2   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        1   1   3   3   2   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        3   1   1   1   2   1   1   1   0      7.68070656e-02
   1      1   1   1   1   0   0   0   0   0        3   3   1   1   2   1   1   1   0      -1.64704540e-01
   1      1   1   3   1   0   0   0   0   0        1   1   1   1   2   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        1   1   3   1   2   1   1   1   0      2.07579903e-01
   1      1   1   3   1   0   0   0   0   0        1   1   3   3   2   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        3   1   1   1   2   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        3   3   1   1   2   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   1   1   2   1   1   1   0      7.68070656e-02
   1      3   1   1   1   0   0   0   0   0        1   1   3   1   2   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   3   3   2   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        3   1   1   1   2   1   1   1   0      1.27686495e-01
   1      3   1   1   1   0   0   0   0   0        3   3   1   1   2   1   1   1   0      -5.08513652e-02
   1      0   0   0   0   0   0   0   1   0        0   0   0   0   0   1   1   0   0      -2.01831100e-01
   1      0   0   0   0   0   0   0   1   0        0   0   2   0   0   1   1   0   0      -0.00000000e+00
   1      0   0   0   0   0   0   0   1   0        2   0   0   0   0   1   1   0   0      2.05509302e-02
   1      0   0   0   0   0   0   0   1   0        0   0   4   0   0   1   1   0   0      -0.00000000e+00
   1      0   0   0   0   0   0   0   1   0        2   0   2   0   0   1   1   0   0      -0.00000000e+00
   1      0   0   0   0   0   0   0   1   0        2   2   2   2   0   1   1   0   0      -0.00000000e+00
   1      0   0   0   0   0   0   0   1   0        4   0   0   0   0   1   1   0   0      4.19919025e-02
```

After:
```
# RELATIVE-CM LSJT
#   version
#   J0 g0 T0_min T0_max symmetry_phase_mode  [P0=(-)^g0]
#   Nmax
#   T0   Nr' lr' Nc' lc' L' S' J' T' g'   Nr lr Nc lc L S J T g   JT-RME
 1
 1 0 1 1 0
 4
   1      1   1   1   1   0   0   0   0   0        1   1   1   1   0   1   1   1   0      -1.98073763e-01
   1      1   1   1   1   0   0   0   0   0        1   1   3   1   0   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        3   1   1   1   0   1   1   1   0      -4.16307108e-02
   1      1   1   3   1   0   0   0   0   0        1   1   1   1   0   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        1   1   3   1   0   1   1   1   0      -1.98073763e-01
   1      1   1   3   1   0   0   0   0   0        3   1   1   1   0   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   1   1   0   1   1   1   0      -4.16307108e-02
   1      3   1   1   1   0   0   0   0   0        1   1   3   1   0   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        3   1   1   1   0   1   1   1   0      -1.14243860e-01
   1      1   1   1   1   0   0   0   0   0        2   2   2   2   1   0   1   1   0      0.00000000e+00  <<<
   1      1   1   3   1   0   0   0   0   0        2   2   2   2   1   0   1   1   0      8.73909393e-02  <<<
   1      3   1   1   1   0   0   0   0   0        2   2   2   2   1   0   1   1   0      0.00000000e+00  <<<
   1      1   1   1   1   0   0   0   0   0        1   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        1   1   3   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        3   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        1   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        1   1   3   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        3   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   3   1   1   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        3   1   1   1   1   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        1   1   1   1   2   1   1   1   0      2.07579903e-01
   1      1   1   1   1   0   0   0   0   0        1   1   3   1   2   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        1   1   3   3   2   1   1   1   0      -0.00000000e+00
   1      1   1   1   1   0   0   0   0   0        3   1   1   1   2   1   1   1   0      7.68070656e-02
   1      1   1   1   1   0   0   0   0   0        3   3   1   1   2   1   1   1   0      -1.64704540e-01
   1      1   1   3   1   0   0   0   0   0        1   1   1   1   2   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        1   1   3   1   2   1   1   1   0      2.07579903e-01
   1      1   1   3   1   0   0   0   0   0        1   1   3   3   2   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        3   1   1   1   2   1   1   1   0      -0.00000000e+00
   1      1   1   3   1   0   0   0   0   0        3   3   1   1   2   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   1   1   2   1   1   1   0      7.68070656e-02
   1      3   1   1   1   0   0   0   0   0        1   1   3   1   2   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        1   1   3   3   2   1   1   1   0      -0.00000000e+00
   1      3   1   1   1   0   0   0   0   0        3   1   1   1   2   1   1   1   0      1.27686495e-01
   1      3   1   1   1   0   0   0   0   0        3   3   1   1   2   1   1   1   0      -5.08513652e-02
   1      0   0   0   0   0   0   0   1   0        0   0   0   0   0   1   1   0   0      -2.01831100e-01
   1      0   0   0   0   0   0   0   1   0        0   0   2   0   0   1   1   0   0      -0.00000000e+00
   1      0   0   0   0   0   0   0   1   0        2   0   0   0   0   1   1   0   0      2.05509302e-02
   1      0   0   0   0   0   0   0   1   0        0   0   4   0   0   1   1   0   0      -0.00000000e+00
   1      0   0   0   0   0   0   0   1   0        2   0   2   0   0   1   1   0   0      -0.00000000e+00
   1      0   0   0   0   0   0   0   1   0        2   2   2   2   0   1   1   0   0      -0.00000000e+00
   1      0   0   0   0   0   0   0   1   0        4   0   0   0   0   1   1   0   0      4.19919025e-02
```

Many of the other matrix elements differ in phase, but this is just the first few.

**Comment by kc9jud** [comment]
The “before” values came from the commit immediately prior.

**Comment by e-eight** [comment]
This is because I changed the phase convention of the 3DHO basis functions to "positive at origin" in e-eight/basis_func@5374e4c034c4d25c019d9338ac9c0ec39929dd70

**Comment by e-eight** [comment]
@kc9jud Does that answer your question?

**Comment by kc9jud** [comment]
@e-eight I'm finally getting back to this -- I'm trying to benchmark against your slides from 2020-04-29 and I can't seem to reproduce the numbers for 3He with the newer version. However, I do get the same numbers if I use the old commit. In addition, in heavier nuclei I get really strange results if I use the newer version. Can you confirm that you're still getting the same numbers as back in those slides?

**Comment by kc9jud** [comment]
Closed by #9.
